translate/api/Tests/TranslateFixture.cs

```diff
@@ -0,0 +1,69 @@
+﻿// Copyright 2019 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Apis.Storage.v1.Data;
+using Google.Cloud.Storage.V1;
+using System;
+using Xunit;
+
+namespace GoogleCloudSamples
+{
+    /// <summary>
+    /// Fixture which is set up at the start of the test run, and torn down at the end.
+    /// This creates new buckets with test data, and deletes them at the end of the test.
+    /// Also, it creates glossary for translation and delete that glossary at the end of test.
+    /// </summary>
+
+    [CollectionDefinition(nameof(TranslateFixture))]
+    public sealed class TranslateFixture : IDisposable, ICollectionFixture<TranslateFixture>
+    {
+        private readonly string _glossaryInputUri = "gs://cloud-samples-data/translation/glossary_ja.csv";
+        private readonly StorageClient _client;
+
+        public readonly string _projectId = Environment.GetEnvironmentVariable("GOOGLE_PROJECT_ID");
```

Comment:
If it is public, which this should be, it's better if it is a property. And if you only set it on declaration or on the class constructor it can be a get only property.
`public string ProjectId { get; } = Environement....;`